domain/consensus/processes/consensusstatemanager/resolve_block_status.go

```diff
@@ -237,3 +297,86 @@ func (csm *consensusStateManager) selectedTip(stagingArea *model.StagingArea) (*
 
 	return virtualGHOSTDAGData.SelectedParent(), nil
 }
+
+func (csm *consensusStateManager) reverseUTXODiffs(stagingArea *model.StagingArea,
+	unverifiedBlocks []*externalapi.DomainHash, tipUTXOSet, oneBlockBeforeTipUTXOSet externalapi.UTXODiff,
```

Comment:
`unverifiedBlocks` aren't necessarily unverified. Maybe we should just call it `chain`?